pom.xml

```diff
@@ -143,44 +143,22 @@
           <artifactId>spring-boot-maven-plugin</artifactId>
           <version>${version.org.springframework.boot}</version>
         </plugin>
-        <!--
-          Disable Jacoco check inherited from kie-parent. Remove this when it's removed from kie-parent
-          (but some projects still use the check fail build).
-        -->
-        <plugin>
-          <groupId>org.jacoco</groupId>
-          <artifactId>jacoco-maven-plugin</artifactId>
-          <executions>
-            <execution>
-              <id>default-check</id>
-              <goals>
-                <goal>check</goal>
-              </goals>
-              <phase>none</phase>
-            </execution>
-          </executions>
-        </plugin>
       </plugins>
     </pluginManagement>
   </build>
 
   <profiles>
     <profile>
+      <!--
+        TODO: For moving the coverage measurement from Travis to Jenkins, add the repository name to
+        SONARCLOUD_ENABLED_REPOSITORIES in
+        https://github.com/kiegroup/kie-jenkins-scripts/blob/master/job-dsls/jobs/pr_jobs.groovy
+      -->
       <id>run-code-coverage</id>
       <properties>
         <jacoco.excludes>*Lexer</jacoco.excludes>
-        <jacoco.exec.file>${project.build.directory}/jacoco.exec</jacoco.exec.file>
         <jacoco.agent.line>-javaagent:${settings.localRepository}/org/jacoco/org.jacoco.agent/${version.jacoco.plugin}/org.jacoco.agent-${version.jacoco.plugin}-runtime.jar=destfile=${jacoco.exec.file},append=true,excludes=${jacoco.excludes}</jacoco.agent.line>
```

Comment:
The agent line can be generated using `prepare-agent` goal.

Reply:
I am not in favour of this change; I prefer the `jacoco.agent.line` value being visible to having it set by the jacoco maven plugin under the hood.

Reply:
The value is logged by the plugin during build so it is visible, isn't it?

> [INFO] --- jacoco-maven-plugin:0.8.5:prepare-agent (jacoco-prepare-agent) @ employee-rostering-backend ---
[INFO] jacoco.agent.argLine set to -javaagent:/home/jlocker/.m2/repository/org/jacoco/org.jacoco.agent/0.8.5/org.jacoco.agent-0.8.5-runtime.jar=destfile=/home/jlocker/src/github.com/kiegroup/optaweb-employee-rostering/employee-rostering-backend/target/jacoco.exec,excludes=*Lexer


 I thought that `prepare-agent` is the intended way to populate the property unless you have some corner case scenario or need to override low-level defaults that cannot be set using the plugin configuration.

I also think it is safer to use `prepare-agent` because if the default content of the agent argLine changes in a newer version and we don't reflect the change in this place, the plugin might stop working correctly, whereas it would most likely keep working if the property was populated using `prepare-agent`.

Please correct me if I'm missing something. Is there a reason to have the value hard-coded here other than visibility? Is the format documented?

Reply:
There are no other reasons than visibility that I am aware of.

The jacoco agent argLine format is documented here:
https://www.eclemma.org/jacoco/trunk/doc/agent.html

> whereas it would most likely keep working if the property was populated using prepare-agent.
I agree this point might overweight the visibility. Let me take a look.

Reply:
Changed to jacoco:prepare-agent